.env

```diff
@@ -0,0 +1 @@
+API_BASE_URL = "https://dev-uba-bap.tekdinext.com/api"
```

Comment:
remove .env from PR and add .env.example as example @sonaliTekdi

Reply:
removed .env from PR